repo2docker/buildpacks/legacy.py

```diff
@@ -1,11 +1,17 @@
-"""
-Generates a variety of Dockerfiles based on an input matrix
+"""Generates legacy Dockerfiles based on an input matrix.
```

Comment:
The 'input matrix' is leftover from a copy-paste when all the buildpacks were in one file. It doesn't apply here.

How about:

> Generates Dockerfiles from the legacy Binder Dockerfiles based on andrewosh/binder-base.
> The Dockerfile is amended to add the contents of the repository to the image and install a supported version of the notebook and IPython kernel.

Reply:
Sounds much better. Thanks @minrk.